src/app/frontend/chrome/chrome.scss

```diff
@@ -22,6 +22,7 @@
 
 .kd-toolbar {
   box-shadow: $whiteframe-shadow-z1;
+  min-height: 64px;
```

Comment:
Didn't expect that to be a feature :). Somehow it makes sense now. For now I'll just set fixed position for the button. This can be changed easly if needed later.